apps/ledger-live-mobile/src/screens/SyncOnboarding/SoftwareChecksStep.tsx

```diff
@@ -376,7 +372,7 @@ const SoftwareChecksStep = ({ device, isDisplayed, onComplete }: Props) => {
                 button: "check if hardware genuine later",
                 drawer: "Failed Stax hardware check",
               });
-              setGenuineCheckStatus("failed");
+              setGenuineCheckStatus("skipped");
```

Comment:
Just above on

```
          <UnlockDeviceDrawer
            isOpen={nextDrawerToDisplay === "unlock-needed"}
            onClose={() => {
              // Closing because the user pressed on close button, and the genuine check is ongoing
              if (genuineCheckStatus === "ongoing") {
                // Fails the genuine check entirely
                setGenuineCheckStatus("failed");
              }
``` 

the `setGenuineCheckStatus("failed");` needs to be set to `skipped` too <- it's if the user does not unlock their device and close the drawer

Reply:
or actually we can keep it to `failed`, so the drawer you added is displayed ! It's actually better this way